Makefile

```diff
@@ -413,6 +413,28 @@ testdata:
 go-mod-tidy:
 	find ./ -type f -name go.mod -execdir go mod tidy \;
 
+.PHONY: update-vmlinux
+update-vmlinux:
+	KERNEL_VERSION=$${KERNEL_VERSION:-6.15.5} && \
+	IMG=$${CI_KERNELS_REPO:-ghcr.io/inspektor-gadget/ci-kernels}:$${KERNEL_VERSION}-debug && \
+	TMP=$$(mktemp -d btfs-tmp.XXXXXXXXXX) && \
+	pushd $$TMP && \
+	skopeo copy --multi-arch all docker://$$IMG oci-archive:image.tar:$$IMG && \
```

Comment:
Since we already use `crane`, I am wondering if this could be achieved by `crane` or we need to use `skopeo`?